draft-ietf-tls-dtls-connection-id.md

```diff
@@ -376,9 +392,9 @@ Importantly, the sequence number makes it possible for a passive attacker
 to correlate packets across CID changes. Thus, even if a client/server pair
 do a rehandshake to change CID, that does not provide much privacy benefit.
 
-This document does not change the security properties of DTLS {{RFC6347}}.
-It merely provides a more robust mechanism for associating an incoming packet
-with a stored security context.
+The CID-enhanced record layer introduces record padding; a privacy feature 
+not available with the original DTLS 1.2 RFC. Padding allows to inflate the 
+size of the ciphertext making traffic analysis more difficult. 
```

Comment:
Just to mention:
For that feature it would make sense, to use the new record type even if the CID is empty.
That's what the issues #43 and the discussion in #22 is about.

As already commented, I have no preference.

Reply:
But we are not going to add extra functionality in DTLS 1.2 when we already have the same features in 1.3